### DISCUSSION TITLE
In which file is the bot configuring?

### DISCUSSION THREAD

**Discussion by Juranik** [n3d1117/chatgpt-telegram-bot#412] [Q&A]
In which file is the bot itself configured? Where can I add additional functions to it? For example, I want the restriction to be not for tokens, but for requests, please tell me where in which file I can remove the restriction on tokens, and make up my own restrictions? And by the way, the restriction does not work at all, I tried, set the user budget to 0.01, the user could still communicate with the bot

**Comment by n3d1117** [comment]
Hey @Juranik, the bot is configured in the `.env` file. Make sure to read the [README](https://github.com/n3d1117/chatgpt-telegram-bot#getting-started) and the [budget manual](https://github.com/n3d1117/chatgpt-telegram-bot/discussions/184). If you have a specific issue, please include additional details like .env file used and detailed description
